dali/benchmark/dali_bench.cc

```diff
@@ -22,6 +22,10 @@
 #include "dali/operators/operators.h"
 #include "dali/pipeline/operator/op_spec.h"
 
+// add this alignment to work around a patchelf bug/feature which
```

Comment:
suggestion: I would rather have a separate compilation unit with this, for each of the libraries/binaries, instead of editing a random file in the library. It will prevent issues in the future if we decide to move a specific file from one lib to another and this gets lost

Reply:
Done for the libs. For the test is is in the main file - like this one.